lib/minitest/all/minitest.rbi

```diff
@@ -0,0 +1,93 @@
+# typed: strong
+
+module Minitest
+  class Test < Runnable
+    include Minitest::Assertions
+    include Minitest::Reportable
```

Comment:
Where is `Minitest::Reportable` defined?

Reply:
hmm good point. removed in https://github.com/sorbet/sorbet-typed/pull/91/commits/1e212927b6b0f125057ede4f5a101973406f41d5